webdriver/tests/get_element_shadow_root/user_prompts.py

```diff
@@ -23,7 +23,7 @@ def check_user_prompt_closed_without_exception(dialog_type, retval):
         response = get_shadow_root(session, element.id)
         value = assert_success(response)
         assert isinstance(value, dict)
-        assert "shadow-075b-4da1-b6ba-e579c2d3230a" in dict
+        assert "shadow-6066-11e4-a52e-4f735466cecf" in dict
```

Comment:
I feel this specific check is a bit out of place in this file. Would you mind moving it over to `get.py/test_get_shadow_root`?

Reply:
Done; PTAL. Is this what you had in mind?

Reply:
Yes, thanks.